app/models/setting/content.rb

```diff
@@ -28,6 +29,9 @@ def self.default_settings
                       nil,
                       collection: proc { http_proxy_select }, include_blank: N_("no global default")
               ),
+      self.set('cdn_ssl_version', N_("SSL version used to communicate with the CDN"),
```

Comment:
The description is the same as the setting name. How about changing the name here to 'CDN SSL version' ?